static/app/views/issueList/overview.tsx

```diff
@@ -636,6 +638,9 @@ class IssueListOverview extends React.Component<Props, State> {
           return;
         }
 
+        if (this.state.undo) {
```

Comment:
would this only fire if they manage to click undo before the next api call finishes?